### PR TITLE
Audiracmichelle/issue24

### PR DESCRIPTION
**Pull request by audiracmichelle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Zip types as described in the UDS mapper site:
 ## Content
 
 **src**
-* [src/create_data_symlinks.py](src/create_data_symlinks.py) Creates data subfolders and symbolic links if indicated in the [datapaths](conf/datapath) config file.
+* [src/create_dir_paths.py](src/create_dir_paths.py) Creates data subfolders and symbolic links if indicated in the [datapaths](conf/datapath) config file.
 * [src/download_uds_xwalks.py](src/download_uds_xwalks.py) Downloads uds xwalks from source url's.
 * [src/create_clean_uds.py](src/create_clean_uds.py) creates a single clean crosswalk from the raw crosswalks.
 * [src/master_xwalk.py](src/master_xwalk.py) applies processing decisions to obtain a single row for each zipcode.
```
Comment (lindadelacombaz) [inline README.md:80]: Somehow I cannot do a suggestion for adding a new line, but I have added the unique_zcta.py file here too.
```suggestion
* [src/master_xwalk.py](src/master_xwalk.py) applies processing decisions to obtain a single row for each zipcode.
* [src/unique_zcta.py](src/unique_zcta.py) Extracts unique ZCTAs from uds xwalks, maps them to census editions and expands them across years.
```

File: src/unique_zcta.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        SELECT 
+            DISTINCT 
+                zcta, 
+                year, 
+                year - (year % 10) AS census_edition
+        FROM
+            'data/intermediate/uds_clean_xwalk/uds_clean_xwalk.csv'
+        ORDER BY
+            zcta, year
+        """).fetchdf()
```
Comment (lindadelacombaz) [inline src/unique_zcta.py]: It is a detail but I have modified these lines that both of the queries have the same workflow ? (like first define the query and then do df = ....fetchdf() ? I do not know if that is relevant or not as it causes not bugs :).

```suggestion
    query = """
        SELECT 
            DISTINCT 
                zcta, 
                year, 
                year - (year % 10) AS census_edition
        FROM
            'data/intermediate/uds_clean_xwalk/uds_clean_xwalk.csv'
        ORDER BY
            zcta, year
    """
    df = conn.execute(query).fetchdf()
```

File: conf/datapaths/cannon_datapaths.yaml [hunk 3]
```diff
@@ -1,10 +1,7 @@
 # if the files are stored in cannon, then the paths are: 
 input:
-  uds_raw_xwalks: null
-  #/n/holyscratch01/dominici_lab/Lab/uds_raw_xwalks
-  #zcta_shapefiles: /n/dominici_lab/lab/data/exposures/exposure/zcta_shapefiles_tigerline
+  uds_raw_xwalks: /n/dominici_lab/lab/lego/geoboundaries/zip2zcta__uds/raw
 intermediate:
-  uds_clean_xwalk: null
-  #/n/holyscratch01/dominici_lab/Lab/uds_clean_xwalk
+  uds_clean_xwalk: /n/dominici_lab/lab/lego/geoboundaries/zip2zcta__uds/.intermediate
```
Comment (lindadelacombaz) [inline conf/datapaths/cannon_datapaths.yaml:5]: I am not familiar with cannon but just a heads up that here there is a dot before intermediate, it's surely intentional but I just prefer to flag it

**Review by lindadelacombaz — CHANGES_REQUESTED** [review]
- I've added the src/unique_zcta.py and a little description in the ##content section in the README file as a suggested commit for you to review
- The others comments are details and opened to be changed or not as they cause not bugs

File: conf/datapaths/cannon_datapaths.yaml (hunk 3 above)
Comment (audiracmichelle) [reply conf/datapaths/cannon_datapaths.yaml:5]: yep! this is intentionally a hidden folder. Thanks for noting!

**Comment by audiracmichelle** [comment]
worked on the suggestions, and requested re-review for merging approval! @lindadelacombaz

**Review by lindadelacombaz — APPROVED** [review]

**Pull request merged**